### ISSUE TITLE
EorroQuestion

### ISSUE THREAD

**Issue by klfxmu** [voxelmorph/voxelmorph#276]
**Task** (what are you trying to do/register?
training model with pytorch

**What have you tried**
just recurrence your work

Please describe specifics of your approach // use of `vxm`
just use gpu to train
loss=mse
data=atlas.npz(i don't know how to set my trainning data one npz?two npz?or more?)

Question:
the code can run normally but the loss=0.
i have examined the input of network and the output of network.
i have not found problems

**Comment by adalca** [comment]
@klfxmu I'm sorry but I can't parse your question. Are you asking why the loss is 0? Can you please show us what command you ran?

Finally, have you gone through the tutorial?

**Comment by klfxmu** [comment]
Thanks for your reply .
Sorry ,i didn't make my self clear .
My training loss is 0.
I have gone through the tutorial.
The command i run : ./train.py /data3/klz/voxelmorph-master/training --model-dir /data3/klz/voxelmorph-master/scripts/torch/models --gpu 2
Now the result changes:
epoch: 0382  step: 100/100   time: 2.46 sec  loss: -0.356006  (-0.356020, 0.000014)
epoch: 0383  step: 1/100     time: 1.47 sec  loss: -0.356006  (-0.356020, 0.000014)
epoch: 0383  step: 2/100     time: 2.47 sec  loss: -0.356007  (-0.356021, 0.000014)

**Comment by adalca** [comment]
so to be clear, the loss doesn't seem to be 0?

Also, can you show an example of an image?

As an example, you should be able to use the data in https://github.com/adalca/medical-datasets/blob/master/neurite-oasis.md
